lib/screen/editorial/editorial_page.dart

```diff
@@ -123,11 +123,14 @@ class EditorialPageState extends State<EditorialPage>
     final theme = Theme.of(context);
     return Scaffold(
       appBar: AppBar(
-        toolbarHeight: 0,
-      ),
+          toolbarHeight: 0,
+          systemOverlayStyle: const SystemUiOverlayStyle(
```

Comment:
this seems to revert the status bar color bugs fixing. (if it's not needed, please revert changes for this page).